drivers/tapdisk-disktype.c

```diff
@@ -182,7 +175,6 @@ extern struct tap_disk tapdisk_llpcache;
 extern struct tap_disk tapdisk_llecache;
 extern struct tap_disk tapdisk_valve;
 extern struct tap_disk tapdisk_nbd;
-extern struct tap_disk tapdisk_ntnx;
 
 const struct tap_disk *tapdisk_disk_drivers[] = {
 	[DISK_TYPE_AIO]         = &tapdisk_aio,
```

Comment:
There are a lot of disk types here that are `#if 0`, I assume these have been deprecated for some time, should they be dropped completely from this list and the above extern declarations? Perhaps when you do the next cleanup PR.